.github/workflows/workflow.yml

```diff
@@ -0,0 +1,12 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./gradlew test
+      - uses: actions/upload-artifact@v2
```

Comment:
We want this step to always run. Currently, if the tests fail, this step will be skipped. See [this snippet](https://github.com/neelkamath/omni-chat/blob/master/.github/workflows/workflow.yml#L42) for a hint.